querybook/migrations/versions/f449a73c5838_add_boardeditors_board_as_board_item.py

```diff
@@ -58,11 +58,33 @@ def upgrade():
         sa.PrimaryKeyConstraint("id"),
         sa.UniqueConstraint("board_id", "uid", name="unique_board_user"),
     )
+
+    op.add_column(
+        "board_item", sa.Column("query_execution_id", sa.Integer(), nullable=True)
+    )
+    op.create_foreign_key(
+        "board_item_query_id_fk",
+        "board_item",
+        "query_execution",
+        ["query_execution_id"],
+        ["id"],
+        ondelete="CASCADE",
+    )
     # ### end Alembic commands ###
 
 
 def downgrade():
     # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint("board_item_query_id_fk", "board_item", type_="foreignkey")
+    op.create_foreign_key(
```

Comment:
this foreign key already exists

Reply:
what do you mean?

Reply:
the migration is above only renames board_id to parent_board_id, so there is already a foreign key relation. This would duplicate the foreign key relationship and cause the downgrade to fail?

Reply:
oh got it i thought it was drop_foreign_key